cfn-resources/serverless-instance/cmd/resource/resource.go

```diff
@@ -0,0 +1,395 @@
+package resource
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/constants"
+	log "github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
+	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
+	mongodbatlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	CallBackSeconds = 30
+)
+
+var CreateRequiredFields = []string{constants.ProjID, constants.PvtKey, constants.PubKey, constants.Name, constants.Name}
+var ReadRequiredFields = []string{constants.ProjID, constants.Name, constants.PvtKey, constants.PubKey}
+var UpdateRequiredFields = []string{constants.PvtKey, constants.PubKey}
+var DeleteRequiredFields = []string{constants.ProjID, constants.Name, constants.PvtKey, constants.PubKey}
+var ListRequiredFields = []string{constants.PvtKey, constants.PubKey}
+
+func validateModel(fields []string, model *Model) *handler.ProgressEvent {
+	return validator.ValidateModel(fields, model)
+}
+
+func setup() {
+	util.SetupLogger("mongodb-atlas-ServerlessInstance")
+}
+
+func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler.ProgressEvent, error) {
+	setup()
+	_, _ = log.Debugf("Create() currentModel:%+v", currentModel)
```

Comment:
```suggestion
	log.Debugf("Create() currentModel:%+v", currentModel)
```

Reply:
This is because of golint error, not sure if this is exempted now.